main.py

```diff
@@ -9,6 +9,14 @@
 from playwright.async_api import async_playwright
 from rich import print
 
+BASE_URL = 'https://prototype-maps.vercel.app'
+DATASETS = {'direct-client': {'v2': ['c2', 'c3', 'c4', 'c5'], 'v3': ['c2', 'c3', 'c4', 'c5']}}
```

Comment:
FYI I just tweaked these names to be more descriptive here: https://github.com/carbonplan/prototype-maps/pull/10

Reply:
thank you for the heads up... i'm going to update these shortly